File: Python/Triangle.py
def is_triangle(side1: float, side2: float, side3: float) -> bool:
    return side1 + side2 > side3 and side1 + side3 > side2 and side2 + side3 > side1

print(is_triangle(3.5, 4.5, 5.5))
print(is_triangle(3, 4, 5))
print(is_triangle(2, 2, 4))
print(is_triangle(-2, 3.1, 4.4))

File: CPP/Triangle.cpp
#include <iostream>

using namespace std;

bool isTriangle(double side1, double side2, double side3) {
    return side1 + side2 > side3 && side1 + side3 > side2 && side2 + side3 > side1;
}



int main() {
    cout << isTriangle(3.5, 4.5, 5.5) << endl;
    cout << isTriangle(3, 4, 5) << endl;
	cout << isTriangle(2, 2, 4) << endl;
	cout << isTriangle(-2, 3.1, 4.4) << endl;
}